src/main/java/org/alliancegenome/curation_api/services/helpers/notes/NoteIdentityHelper.java

```diff
@@ -0,0 +1,33 @@
+package org.alliancegenome.curation_api.services.helpers.notes;
+
+import java.util.Collections;
+import java.util.List;
+
+import javax.enterprise.context.RequestScoped;
+
+import org.alliancegenome.curation_api.model.ingest.dto.NoteDTO;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang.StringUtils;
+
+@RequestScoped
```

Comment:
If the class is going to be statically referenced then it doesn't need the @RequestScoped